src/main/java/stsjorbsmod/potions/GhostPoisonPotion.java

```diff
@@ -0,0 +1,78 @@
+package stsjorbsmod.potions;
+
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.PowerTip;
+import com.megacrit.cardcrawl.localization.PotionStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.potions.AbstractPotion;
+import com.megacrit.cardcrawl.powers.BackAttackPower;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+import com.megacrit.cardcrawl.vfx.combat.SmokeBombEffect;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.actions.IncreaseManifestAction;
+
+public class GhostPoisonPotion extends AbstractPotion {
+    public static final String POTION_ID = JorbsMod.makeID(GhostPoisonPotion.class.getSimpleName());
+    private static final PotionStrings potionStrings = CardCrawlGame.languagePack.getPotionString(POTION_ID);
+    public static final String NAME = potionStrings.NAME;
+    public static final String[] DESCRIPTIONS = potionStrings.DESCRIPTIONS;
+
+    private static final int BASE_POTENCY = 1;
+
+    public GhostPoisonPotion() {
+        super(NAME, POTION_ID, PotionRarity.COMMON, PotionSize.H, PotionColor.SMOKE);
+        this.potency = this.getPotency();
+        this.description = String.format(DESCRIPTIONS[0], this.potency);
+        this.isThrown = true;
+        this.tips.add(new PowerTip(this.name, this.description));
```

Comment:
Consider also adding a tip for the manifest keyword

Reply:
I'm not sure how to reference a keyword for these purposes, since they're not a class. Considering other potions don't clarify using tips, I hope that's ok for now.